tests/para-tests-no-ci/test-xcm-para.ts

```diff
@@ -114,7 +114,7 @@ describeParachain(
 
       // check asset in storage
       const registeredAsset = await parachainOne.query.assets.asset(assetId);
-      expect((registeredAsset.toHuman() as { owner: string }).owner).to.eq(palletId.toLowerCase());
+      expect((registeredAsset.toHuman() as { owner: string }).owner).to.eq(palletId);
```

Comment:
This is a change that was not fixed when we merged the Account20 PR

Reply:
Can we also get rid of toHuman? I remember Jaco discouraged our using that in tests.

Reply:
sure, will try